src/registration/CommunicationPreferencesCheckbox.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { LabeledCheckbox } from 'src/components/common';
+
+interface CommunicationPreferencesCheckboxProps {
+  title: string;
+  value: boolean;
+  setFunc: React.Dispatch<React.SetStateAction<boolean>> | undefined;
```

Comment:
This component doesn't need to require a setState function.

Also, suggest `prop?: Type` for optional props instead of `prop: Type | undefined`. This allows omitting the prop altogether.

```suggestion
  onChange?: (value: boolean) => void;
```